modules/account-lib/test/unit/coin/sol/transactionBuilder/stakingActivateBuilder.ts

```diff
@@ -0,0 +1,275 @@
+// import should from 'should';
+import should from 'should';
```

Comment:
Delete comment

Reply:
changed locally, waiting for other comments to upload this.